src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public int getNumber() {
+        return Lottery.getRandomBall().number;
+    }

```

Comment:
getter should return only own fields
```suggestion
    public int getNumber() {
        return number;
    }

```